### PR TITLE
:rocket: working example with aws inspector add

### PR DESCRIPTION
**Pull request by hernanku**

### CODE REVIEW

**Review by mayank0202 — APPROVED** [review]

File: example/dev.tfvars [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+security_hub_sns_subscribers = {
+  opsgenie = {
+    protocol               = "https"
+    endpoint               = ""
+    endpoint_auto_confirms = true
+    raw_message_delivery   = false
+  }
+}
+enabled_standards = ["standards/aws-foundational-security-best-practices/v/1.0.0", "ruleset/cis-aws-foundations-benchmark/v/1.2.0"]
```
Comment (jamescrowley321) [inline example/dev.tfvars:49]: Upgrade to lastest CIS which is 1.4

File: example/main.tf [hunk 2]
```diff
@@ -2,16 +2,42 @@
 ## defaults
 ################################################################################
 terraform {
-  required_version = "~> 1.3, < 2.0.0"
+  required_version = ">= 1.0"
```
Comment (jamescrowley321) [inline example/main.tf:5]: Can we move this up to the latest version since we won't be supporting anything lower than 1.5?

File: example/main.tf [hunk 3]
```diff
@@ -2,16 +2,42 @@
...
 
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.0"
+      version = ">= 2"
```
Comment (jamescrowley321) [inline example/main.tf:10]: Bump up to 4 as 2 and 3 may not work.

File: example/main.tf [hunk 4]
```diff
@@ -2,16 +2,42 @@
...
     }
   }
 }
-
 provider "aws" {
   region = var.region
 }
+
+module "cloud_security" {
+  # source      = "git::https://github.com/sourcefuse/terraform-aws-arc-security.git?ref=feature/working-example"
```
Comment (jamescrowley321) [inline example/main.tf:19]: Remove commented line

File: main.tf [hunk 5]
```diff
@@ -1,13 +1,99 @@
-################################################################################
-## defaults
-################################################################################
 terraform {
-  required_version = "~> 1.3, < 2.0.0"
+  required_version = ">= 1.0"
```
Comment (jamescrowley321) [inline main.tf:2]: Same comments here as above

**Review by jamescrowley321 — CHANGES_REQUESTED** [review]

File: example/main.tf (hunk 2 above)
Comment (hernanku) [reply example/main.tf:5]: I tried, but it seems the restriction is coming from the underlying cloudposse module
image

https://github.com/cloudposse/terraform-aws-security-hub

We could fork and update

File: example/main.tf (hunk 3 above)
Comment (hernanku) [reply example/main.tf:10]: Same as above

File: example/main.tf (hunk 4 above)
Comment (hernanku) [reply example/main.tf:19]: Sure, will do. My miss

File: main.tf (hunk 5 above)
Comment (hernanku) [reply main.tf:2]: Same as above

**Review by jamescrowley321 — APPROVED** [review]

**Pull request merged**